bc_obps/registration/api/user_operator.py

```diff
@@ -202,7 +202,6 @@ def request_access(request, payload: SelectOperatorIn):
 @router.post("/user-operator/operator", response={200: RequestAccessOut, codes_4xx: Message})
 def create_operator_and_user_operator(request, payload: UserOperatorOperatorIn):
     user: User = request.current_user
-    raise_401_if_role_not_authorized(user.app_role, ["industry_user", "industry_user_admin"])
```

Comment:
I think this needs to stay in--every endpoint should have role protection in case someone without a role tries to access. If everyone needs to see it, the array can be ["industry_user", "industry_user_admin","cas_admin", "cas_analyst","cas_pending"] (although I can't think why cas_pending would get access to this)

Reply:
re-added